hera_mc/cm_utils.py

```diff
@@ -14,7 +14,8 @@
 import os.path
 
 PAST_DATE = '2000-01-01'
-FUTURE_DATE = '2025-12-31'
+def _future_date():
+    return Time.now() + TimeDelta(100, format='jd')
```

Comment:
What's the motivation for this change? Not obvious to me that there was anything wrong with the previous version.